queries/sql/highlights.scm

```diff
@@ -160,18 +163,54 @@
   (keyword_like)
   (keyword_similar)
   (keyword_over)
+  (keyword_change)
+  (keyword_modify)
+  (keyword_restrict)
+  (keyword_first)
+  (keyword_after)
+  (keyword_range)
+  (keyword_rows)
+  (keyword_groups)
+  (keyword_unbounded)
+  (keyword_preceding)
+  (keyword_following)
+  (keyword_exclude)
+  (keyword_current)
+  (keyword_ties)
+  (keyword_others)
+  (keyword_unique)
+  (keyword_preserve)
+  (keyword_zerofill)
+  (keyword_format)
+  (keyword_fields)
+  (keyword_row)
+  (keyword_sort)
+  (keyword_compute)
+  (keyword_stats)
+  (keyword_comment)
+  (keyword_partitioned)
+  (keyword_location)
+  (keyword_cached)
+  (keyword_lines)
+  (keyword_stored)
 ] @keyword
```

Comment:
It feels like some of these could be `@type.qualifier`, e.g. in C you can have `int* restrict b;` where restrict would be `@type.qualifier`

Reply:
That sounds like a good idea. I will test that out. Sometimes it is hard to find a reasonable pattern in SQL keywords

Reply:
Anything that modifies how a variable/item behaves at compile time or runtime fits under `@type.qualifier`. Some more examples could be abstract, const, static, dynamic etc..

Reply:
> Sometimes it is hard to find a reasonable pattern in SQL keywords

I agree, SQL has so many different keywords it's hard to sort through them all.

Reply:
I added these as `@type.qualifier` and some others as `@attribute` From my perspective they are now grouped in a reasonable manner

Reply:
cached, location, partitioned, and maybe computer might seem like they fit in `@storageclass`, this is usually for qualifiers or attributes that affect lifetime or memory of whatever

Reply:
These keywords all belong to hive specific statements:
https://impala.apache.org/docs/build/html/topics/impala_create_table.html

I would rather argue that they should be moved back to be a normal `@keyword` as they start each line of these create table statements

Some of the `@attributes` are maybe better of as a `@storageclass`
e.g. `keyword_materialized`, `keyword_temp, `keyword_logged` and the file formats

Reply:
can't speak on materialized or logged, but temp does seem to fit, and if the other 2 are related then that as well